modules/sipcapture/sipcapture.c

```diff
@@ -4314,7 +4315,7 @@ static int w_hep_relay(struct sip_msg *msg)
 		&uri.host,
 		uri.port_no?uri.port_no:SIP_PORT, proto, 0 );
 	if (proxy == 0) {
-		LM_ERR("bad host name in URI <%.*s>\n", uri_s->len, ZSW(uri_s->s));
+		LM_ERR("bad host name in URI <%.*s>\n", uri_s->len, redact_pii(uri_s->s));
```

Comment:
If I'm reading this right this is the HEP URI which would most likely be internal but even if external it is defined in the routing script. I'm not sure it needs to be redacted unless someone has a use case where a customer can configure an external HEP endpoint.